packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/library/services/LibraryApplicationService.java

```diff
@@ -45,4 +47,12 @@ public LibraryApplicationService(ILibrarySearchService librarySearchService, ILi
     public Page<LibraryDTO> findAll(Pageable pageable) {
         return this.librarySearchService.findAll(pageable).map(this.libraryMapper::toDTO);
     }
+
+    @Override
+    @Transactional(readOnly = true)
+    public Optional<Library> findByNamespaceAndNameAndVersion(String namespace, String name, String version) {
```

Comment:
The application service is in charge of the creation of the DTO, look at the method above

Reply:
Done